dags/vds_pull_vdsdata.py

```diff
@@ -48,43 +37,52 @@
     'catchup': True,
 }
```

Comment:
I think it might be beneficial to start packing this block of code (updating Python path, setting DAG's default args, getting the doc_md, etc.) which is shared in almost all DAGs in one initialization function for DAGs. and maybe also add it to the Contributing.md file

What do you think? Maybe a new issue and PR for this

Reply:
Created a new issue for this!